### PR TITLE
Add Slowlog fileset for the Elasticsearch module

### PR DESCRIPTION
**Pull request by radoondas**
This is the initial PR for slowlog indexing

### CODE REVIEW

File: CHANGELOG.asciidoc [hunk 1]
```diff
@@ -205,6 +205,7 @@ https://github.com/elastic/beats/compare/v6.2.3...master[Check the HEAD diff]
 - Converted part of pipeline from treafik/access metricSet to dissect to improve efficeny. {pull}7209[7209]
 - Add GC fileset to the Elasticsearch module. {pull}7305[7305]
 - Add Audit log fileset to the Elasticsearch module. {pull}7365[7365]
+- Add Slow log fileset to the Elasticsearch module.
```
Comment (ruflin) [inline CHANGELOG.asciidoc]: Can you link the PR?

File: filebeat/module/elasticsearch/slowlog/_meta/fields.yml [hunk 2]
```diff
@@ -0,0 +1,61 @@
+- name: slowlog
+  description: "Slowlog events from Elasticsearch"
+  example: "[2018-06-29T10:06:14,933][INFO ][index.search.slowlog.query] [v_VJhjV] [metricbeat-6.3.0-2018.06.26][0] took[4.5ms], took_millis[4], total_hits[19435], types[], stats[], search_type[QUERY_THEN_FETCH], total_shards[1], source[{\"query\":{\"match_all\":{\"boost\":1.0}}}],"
+  type: group
+  fields:
+  - name: loglevel
```
Comment (ruflin) [inline filebeat/module/elasticsearch/slowlog/_meta/fields.yml]: This should to under `log.level`. Already exists so does not have to be added.

File: filebeat/module/elasticsearch/slowlog/_meta/fields.yml [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+    description: "Log level"
+    example: "INFO"
+    type: keyword
+  - name: logger
+    description: "Logger name"
+    example: "index.search.slowlog.fetch"
+    type: keyword
+  - name: node_name
```
Comment (ruflin) [inline filebeat/module/elasticsearch/slowlog/_meta/fields.yml]: `elasticsearch.node.name`

File: filebeat/module/elasticsearch/slowlog/_meta/fields.yml [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+    description: "Name of the node"
+    example: "v_VJhjV"
+    type: keyword
+  - name: index_name
```
Comment (ruflin) [inline filebeat/module/elasticsearch/slowlog/_meta/fields.yml]: `elasticsearch.index.name`

We need to check if these two fields already exist or have to be added in Filebeat.

File: filebeat/module/elasticsearch/slowlog/_meta/fields.yml [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+    description: "Name of the index"
+    example: "metricbeat-6.3.0-2018.06.26"
+    type: keyword
+  - name: shard_id
```
Comment (ruflin) [inline filebeat/module/elasticsearch/slowlog/_meta/fields.yml]: I think `elasticsearch.shard.id` (need to check Metricbeat)

File: filebeat/module/elasticsearch/slowlog/_meta/fields.yml [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+    description: "Id of the shard"
+    example: "0"
+    type: keyword
+  - name: took
+    description: "Time it took to execute the query"
+    example: "300ms"
```
Comment (ruflin) [inline filebeat/module/elasticsearch/slowlog/_meta/fields.yml]: Is this always in `ms`, meaning we could extract the value?

File: filebeat/module/elasticsearch/slowlog/_meta/fields.yml [hunk 7]
```diff
@@ -0,0 +1,61 @@
...
+    type: keyword
+  - name: source_query
+    description: "Slow query"
+    example: "{\"query\":{\"match_all\":{\"boost\":1.0}}}"
+    type: text
+  - name: extra_source
+    description: "Extra source information"
+    example: ""
+    type: text
+  - name: took_millis
```
Comment (ruflin) [inline filebeat/module/elasticsearch/slowlog/_meta/fields.yml]: Is this the same as `took`?

File: filebeat/module/elasticsearch/slowlog/_meta/fields.yml [hunk 8]
```diff
@@ -0,0 +1,61 @@
...
+    description: "Time took in milliseconds"
+    example: 42
+    type: keyword
+  - name: total_hits
```
Comment (ruflin) [inline filebeat/module/elasticsearch/slowlog/_meta/fields.yml]: `total.hits`
`total.shards`

Or hits.total and shards.total? Not sure TBH

File: filebeat/module/elasticsearch/slowlog/manifest.yml [hunk 9]
```diff
@@ -0,0 +1,13 @@
+module_version: 1.0
+
+var:
+  - name: paths
+    default:
+      - /var/log/elasticsearch/*_index_search_slowlog.log
+    os.darwin:
+      - /usr/local/elasticsearch/*_index_search_slowlog.log
```
Comment (ruflin) [inline filebeat/module/elasticsearch/slowlog/manifest.yml]: What is the value of the `*` here? Does it contain info like cluster id or node id?

File: CHANGELOG.asciidoc (hunk 1 above)
Comment (radoondas) [reply CHANGELOG.asciidoc]: Yes, already prepared for commit.

**Pull request merged**